### ISSUE TITLE
folder structure, if on git.ignore

### ISSUE THREAD

**Issue by oSamTo** [NERC-CEH/ukem#11]
the function `aggregate_3way()`, inside targets `l_fname_3way_n2o` / `l_fname_3way_ch4` / `l_fname_3way_co2` , has a reference to folder `/output/` within it. Writing of emissions stacks, however, go to `/data/output/`, as per `writeOutput()`

i can see you have a new folder '/output/' inside git.ignore on the Master - i am sure this is why the targets fail for me as i dont have this folder created. Do you know if there's a way of ensuring a user has the required file structure that matches git.ignore?

or perhaps we need `dir.create()` built in to some functions?

**Comment by oSamTo** [comment]
yes, `aggregate_3way()` will not work on the folder structure nominated in the function - it didn't exist before so can't write to it. simple as that really. Needs a `dir.create()` implementing.

**Comment by peterlevy** [comment]
Ok, so we want git to not ignore the folders, just most of the the contents.  I guess there is a .gitignore permutation for that.

**Comment by oSamTo** [comment]
perfect, that sounds a good fix if it can be implemented, i.e. have the folder structure available for people to pull, but nothings actually in it.
